cmake/FindCasacore.cmake

```diff
@@ -83,7 +83,9 @@ if (NOT CASACORE_FOUND)
     ## Search for the library
     find_library (CASACORE_${_casacore_var}_LIBRARY
       NAMES casa_${_casacore_lib} ${_casacore_lib}
-      HINTS ${CASACORE_ROOT_DIR}
+      HINTS
+      ${CASACORE_ROOT_DIR}
+      "/usr/local/casacore"
```

Comment:
Not really necessary if you were already pointing to casacore manually. I'd opt for removing this and have users make this pointing more explicitly in the `cmake` invocation. Maybe a section in the README file about the most useful cmake variables could have this one listed.